app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -121,18 +121,28 @@ class MainActivity : AppCompatActivity() {
     private fun loadFragment(fragment: Fragment) {
         supportFragmentManager.beginTransaction()
                 .replace(R.id.frameContainer, fragment, fragment::class.java.name)
-                .addToBackStack(null)
                 .commit()
     }
 
     override fun onBackPressed() {
-        val currentFragment = this.supportFragmentManager.findFragmentById(R.id.frameContainer)
+
```

Comment:
extra feed

Reply:
Not extra... Here I separate methods for root container and frame container.

Reply:
No Kotlin style guide tells you to add a line feed at the start of method body